starlight.sh

```diff
@@ -0,0 +1,42 @@
+package: STARlight
+version: "%(tag_basename)s"
+tag: master
+build_requires:
+  - CMake
+  - "GCC-Toolchain:(?!osx)"
+source: https://github.com/alisw/STARlight.git
+---
+#!/bin/bash -ex
+cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX=$INSTALLROOT       \
+                 ${CMAKE_GENERATOR:+-G "$CMAKE_GENERATOR"} \
+                 -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE      \
+                 -DCMAKE_SKIP_RPATH=TRUE
+cmake --build . -- ${JOBS:+-j$JOBS}
+mkdir -p $INSTALLROOT/bin
+cp ./starlight $INSTALLROOT/bin/.
+mkdir -p $INSTALLROOT/lib
+cp ./libStarlib.a $INSTALLROOT/lib/.
+cp -r $SOURCEDIR/config $INSTALLROOT
+
+#ModuleFile
+mkdir -p etc/modulefiles
+cat > etc/modulefiles/$PKGNAME <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0                                                                                \\
+
+# Delphes environment:
+set STARLIGHT_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+setenv STARLIGHT_ROOT \$STARLIGHT_ROOT
```

Comment:
Please drop this.

Reply:
hi @ktf 
ok, I will drop it for the moment. But I am not sure it will be needed in the future if it emerges that the location of the library is needed for some reason.
In case I will open a new PR.

Reply:
no, sorry.
I already found a place where I need to link the {{STARlight}} library.
Please, hold on a sec and I check if I can actually deal with it without knowing the location of it.